Source/WebCore/loader/appcache/DOMApplicationCache.idl

```diff
@@ -29,8 +29,7 @@
     DoNotCheckConstants,
     GenerateIsReachable=ReachableFromDOMWindow,
     InterfaceName=ApplicationCache,
-    Exposed=Window,
-    EnabledByQuirk=shouldEnableApplicationCache
```

Comment:
Doesn't this end up always enabling this interface? Similar issue below with the `applicationCache` getter.

I think if you remove the quirk that sometimes enable this you can always remove this entire file.